sr_utilities_common/src/sr_utilities_common/aws_manager.py

```diff
@@ -61,13 +61,26 @@ def __init__(self):
             aws_session_token=aws_session_token
         )
 
-    def get_bucket_structure_with_prefix(self, bucket_name, prefix):
-        if prefix:
-            try:
-                if 'Contents' in self._client.list_objects(Bucket=bucket_name, Prefix=prefix):
-                    return self._client.list_objects(Bucket=bucket_name, Prefix=prefix)['Contents']
-            except self.client.exceptions.NoSuchBucket as e:
-                rospy.logerr("Failed listing bucket objects. " + str(e))
+    def __init__(self, access_key, secret_key, session_token):
+        self.file_full_paths = []
```

Comment:
I think these should be private

Reply:
self.file_full_paths and self.aws_paths 
Sorry for the wrong line points. If this should not be accessible (editable) from outside the class then keep it private

Reply:
They are edited outside the class which is why I made them public